shell/store/github.js

```diff
@@ -242,11 +98,15 @@ export const actions = {
 export const mutations = {
```

Comment:
I'm a bit split on if we should include this. There are no `getters` so they're not used anywhere and the result of the action is just returned. Maybe we just remove them, plus the state. Then we won't have to worry about resetting the state when logging out